Library/SubscriptionChange.cs

```diff
@@ -24,6 +24,12 @@ public enum ChangeTimeframe : short
 
         public string PlanCode { get; set; }
 
+        /// <summary>
+        /// Proration settings to be applied to the subscription change.
+        /// If not set, the default proration behavior will apply.
+        /// </summary>
+        public ProrationSettings ProrationSettings { get; set; }
```

Comment:
should this be nullable?

Reply:
Yes. The user can fallback to default behavior set at the site level or recurly's default behavior which is simply to prorate.

Reply:
Then you want to do something like this

`public ProrationSettings? ProrationSettings { get; set; }`

Reply:
On second thought, this is just an empty object if a user instantiates this class. So no it shouldbe be nullable. What is nullable is Charge and Credit properties.

Reply:
I thought about this but with our version of C# it throws errors both when building the branch and testing building the client locally with the following error: `Feature 'nullable reference types' is not available in C# 7.3.`

I was able to update Recurly.csproj to the latest language version to allow nullable reference types and build successfully but I don't want to change our current configuration.

Reply:
hmm thats really strange there are lots of nullable reference types all over the codebase, you shouldnt need to update the language version

Reply:
Although, looking at this you probably just dont need to make this nullable because everything inside the model itself is nullable, so this is fine

Reply:
@btruncali1 @smagdicatrecurly My understanding is that, because this is a reference type, it's nullable by default - compare dto value types (e.g. `int`), where you would have to declare nullability.

This thinking seems consistent with both the aforementioned compile error and being able to do `ProrationSettings != null` below.